tuxemon/core/states/items/__init__.py

```diff
@@ -83,40 +83,21 @@ def use_item(menu_item):
             player = self.game.player1
             monster = menu_item.game_object
 
```

Comment:
basically, move the combat checking logic to the combat menu.  make the items menu more simple.